### ISSUE TITLE
Combining multiple columns while keeping the column headers

### ISSUE THREAD

**Issue by Stadly** [DataTables/DataTables#825]
It would be great with a possibility to combine the values of multiple consecutive columns into a single column, while keeping the column headers and sorting options. For example, names are easier to read when being displayed like `{Firstname} {Surname}` in one column, rather than being in two columns. But by combining the values, you miss the possibility to sort by both `Firstname` and by `Surname` in `DataTables`.

My request is that, through some smart initialization options, `DataTables` could render a table with separate information in each column:

    <table>
      <thead>
        <tr>
          <th>Firstname</th>
          <th>Surname</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>Donna</td>
          <td>Summer</td>
        </tr>
        <tr>
          <td>Quincy</td>
          <td>Jones</td>
        </tr>
      </tbody>
    </table>

as a table where the information from multiple columns are combined in a single column:

    <table>
      <thead>
        <tr>
          <th class="sorting">Firstname</th>
          <th class="sorting">Surname</th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td colspan="2">Donna Summer</td>
        </tr>
        <tr>
          <td colspan="2">Quincy Jones</td>
        </tr>
      </tbody>
    </table>

In other words, `DataTables` should render the two columns into a `colspan="2"`-column for the rows in `<tbody>` but not in `<thead>`.

In the above example, the table could then be ordered by `Firstname`, where Donna Summer would come before Quincy Jones, while ordering by `Surname`, would show Quincy Jones before Donna Summer.

**Comment by DataTables** [comment]
Its a really nice idea - thanks! However, `colspan` and `rowspan` are not supported in the table body at this time and to be honest, I can't see that chance any time in the short or medium term. That would be required for this to work, so I'm afraid I don't think this is likely to happen.

**Comment by stoem** [comment]
I would like to +1 this. 
Maybe there is a way to do away with colspans by using a single cell to display the firstname and surname and have 2 sorting options in the header that somehow are 'intelligent' enough to sort on either first or surname when clicked?
Just thinking out loud.
